lib/model/query/datasets.js

```diff
@@ -161,11 +164,13 @@ const getById = (datasetId) => ({ all }) =>
     .then(asArray)
     .then(nth(0));
 
+// Returns only published dataset with its published properties
 const getByName = (datasetName, projectId) => ({ all }) =>
```

Comment:
`Datasets.getByName` is called from a single place and it expects to receive only published dataset with its published properties